.github/workflows/trigger.yml

```diff
@@ -0,0 +1,15 @@
+name: Push Images
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  # Allow manually triggering in Actions tab
+  workflow_dispatch:
+
+jobs:
+  image-scan:
+    uses: ./.github/workflows/image-scan.yml
+    with:
+      images: "gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1 docker.io/portworx/pds-mysql-router:8.0.30-676b5f0"
```

Comment:
qq: Any reason for these 2 images specifically? Was this workflow just for testing?